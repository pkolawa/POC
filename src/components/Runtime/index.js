import React from "react";
import Typography from "@material-ui/core/Typography";

class Runtime extends React.Component {
  render() {
    return (
      <div>
        <Typography align="center" variant="h2" color="secondary">
        Runtime Page
        </Typography>
      </div>
    );
  }
}

export { Runtime as default }