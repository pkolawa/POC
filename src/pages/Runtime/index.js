import React from "react";
import { Helmet } from "react-helmet";
import RuntimeManagementContainer from "../../containers/RuntimeManagementContainer";

class Runtime extends React.Component {
  constructor(props) {
    super(props);
  }
  render() {
    return (
      <>
        <Helmet>
          <title>Runtime Management</title>
          <meta
            name="description"
            content="Runtime Management"
          />
        </Helmet>
        <RuntimeManagementContainer />
      </>
    );
  }
}

export default Runtime;
