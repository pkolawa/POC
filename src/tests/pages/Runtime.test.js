import React from "react";
import { shallow } from "enzyme";
import Runtime from "../../pages/Runtime";

test("Should Render Configuration Page", () => {
  const wrapper = shallow(<Runtime/>);
  expect(wrapper).toMatchSnapshot();
});