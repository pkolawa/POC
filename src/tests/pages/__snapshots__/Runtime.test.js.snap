// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Configuration Page 1`] = `
<Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Runtime Management
    </title>
    <meta
      content="Runtime Management"
      name="description"
    />
  </HelmetWrapper>
  <Connect(WithStyles(RuntimeManagementContainer)) />
</Fragment>
`;
